package.json

```diff
@@ -26,9 +31,9 @@
     "build:dist": "node ./scripts/build.js ./lib/browser.js ./LICENSE.md ./dist/twilio.js",
     "build:dist-min": "uglifyjs ./dist/twilio.js -o ./dist/twilio.min.js --comments \"/^! twilio-voice.js/\" -b beautify=false,ascii_only=true",
     "build:release": "npm-run-all lint build test:es5 status",
-    "build:ts": "./node_modules/typescript/bin/tsc",
+    "build:ts": "./node_modules/typescript/bin/tsc && ./node_modules/typescript/bin/tsc --project tsconfig-mjs.json",
```

Comment:
Since we're now producing mjs folder
- we need to update `.release.json` to also include the folder in the built artifacts.
- should we have a `test:mjs` npm script? See `test:es5`. It validates the es5 files. If it's possible and if you agree, we should also call `test:mjs` wherever we call `test:es5` in the package.json and circileci's config.yml.